test/specs/rules/disallow-space-before-semicolon.js

```diff
@@ -0,0 +1,31 @@
+var Checker = require('../../../lib/checker');
+var assert = require('assert');
+
+describe('rules/disallow-space-before-semicolon', function() {
+    var checker;
+
+    beforeEach(function() {
+        checker = new Checker();
+        checker.registerDefaultRules();
+    });
+
+    it('should not accept a false value as an option', function() {
+        assert.throws(function() {
+            checker.configure({ disallowSpaceBeforeSemicolon: false });
+        });
+    });
+
+    it('does not allow spaces before semicolons', function() {
+        checker.configure({ disallowSpaceBeforeSemicolon: true });
+
+        assert(!checker.checkString('; ;').isEmpty());
+        assert(!checker.checkString('var a = 1 ;').isEmpty());
+        assert(!checker.checkString('var a = 2  ;').isEmpty());
```

Comment:
Just missed the `!` :-), but yeah, specific is better, although if we would be real sticklers we would ask you to put assertions in their very own `test` functions since it easier to isolated them with `it.only`.